### PR TITLE
Eng 8680: Fixing txnid-selfcheck2 failing test

### PR DESCRIPTION
**Pull request by JadenW**
manually tested via run.sh and app_runner of 3 cluster for duration of 360s.

### CODE REVIEW

File: tests/test_apps/txnid-selfcheck2/src/txnIdSelfCheck/LoadTableLoader.java [hunk 1]
```diff
@@ -278,15 +279,15 @@ public void run() {
...
+                CountDownLatch latch = new CountDownLatch(batchSize + batchSize * upsertHitMode);
                 final ArrayList<Long> lcpDelQueue = new ArrayList<Long>();
 
                 // try to insert batchSize random rows
                 for (int i = 0; i < batchSize; i++) {
                     m_table.clearRowData();
                     m_permits.acquire();
-                    long p = Math.abs(r.nextLong());
-                    m_table.addRow(p, p, Calendar.getInstance().getTimeInMillis());
+                    m_table.addRow(p, p + Calendar.getInstance().getTimeInMillis(), Calendar.getInstance().getTimeInMillis());
```
Comment (akhanzode) [inline tests/test_apps/txnid-selfcheck2/src/txnIdSelfCheck/LoadTableLoader.java]: Use System.nanotime()

File: tests/test_apps/txnid-selfcheck2/src/txnIdSelfCheck/LoadTableLoader.java [hunk 2]
```diff
@@ -271,6 +271,7 @@ public void run() {
 
         CopyAndDeleteDataTask cdtask = new CopyAndDeleteDataTask();
         cdtask.start();
+        long p = 0;
```
Comment (akhanzode) [inline tests/test_apps/txnid-selfcheck2/src/txnIdSelfCheck/LoadTableLoader.java]: Start with System.nanoTime()

**Comment by akhanzode** [comment]
remove unused imports.

**Comment by prosegay** [comment]
make sure we squash merge this when it is ready

**Comment by akhanzode** [comment]
squash merged.

**Pull request closed without merging**